config/jobs/kubernetes/sig-network/dualstack-e2e.yaml

```diff
@@ -0,0 +1,75 @@
+presets:
+- labels:
+    preset-cloudprovider-azure-cred: "true"
```

Comment:
these are usually in prow/config.yaml I think

Reply:
is this why the job is failing?
```
Could not load config: duplicated preset 'label:value' pair : preset-cloudprovider-azure-cred:true
```

Reply:
It was complaining about the label being used in another job, so switched to using a new label for this job.